biometricks/src/main/java/com/handstandsam/biometricks/internal/BiometricksHelper.kt

```diff
@@ -78,4 +86,34 @@ internal class BiometricksHelper(context: Context) {
             }
         }
     }
+
+    /**
+     * Check if the user can securely authenticate with biometrics.
+     */
+    fun canSecurelyAuthenticate(): Boolean {
```

Comment:
Interesting @evant .  This will only work pre-28 though... what do we do about 28 and 29?

Reply:
It should work on all versions, unless you see something I'm not?

Reply:
On 28 and 29... if you determined it only had fingerprint... you could do the same... but when you have gaps for other devices, what do we do?  I like it, but don't like the fact that we have a large percentage of cases not covered by this.